src/App.js

```diff
@@ -1386,6 +1387,30 @@ export class App extends React.Component<AppProps, AppState> {
             });
         }
 
+        // Keyboard settings are read from local storage whether or not we have URL content.
+        const localKeyBindingsEnabled = window.localStorage.getItem('c2lc-keyBindingsEnabled');
+        const localKeyboardInputSchemeName = window.localStorage.getItem('c2lc-keyboardInputSchemeName');
+
+        if (localKeyBindingsEnabled !== null) {
```

Comment:
I think in this case that it's a little safer to use the less strict '!=' as it will catch `undefined`:

```
> undefined !== null
true
> undefined != null
false
```

Reply:
Good point, and fixed.